api/HTMLCanvasElement.json

```diff
@@ -1333,6 +1333,54 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "type_webp_parameter": {
+          "__compat": {
+            "description": "<code>type</code> parameter <code>image/webp</code>",
+            "support": {
+              "chrome": {
+                "version_added": "17"
+              },
+              "chrome_android": {
+                "version_added": "18"
+              },
+              "edge": {
+                "version_added": "79"
+              },
+              "firefox": {
+                "version_added": "96"
+              },
+              "firefox_android": {
+                "version_added": "96"
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": true
+              },
+              "opera_android": {
+                "version_added": true
+              },
+              "safari": {
+                "version_added": null
+              },
+              "safari_ios": {
+                "version_added": null
+              },
+              "samsunginternet_android": {
+                "version_added": null
+              },
+              "webview_android": {
+                "version_added": null
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
```

Comment:
Not sure about this. The standard says nothing about the use of particular image formats other than png